README.md

```diff
@@ -81,10 +81,11 @@ Details of the available fields in the response are in the Companies House
 [documentation](https://developer.companieshouse.gov.uk/api/docs/index.html).
 The endpoints currently implemented by the gem are:
 
-| Client Method               | Endpoint                                | Description |
-| --------------------------- | --------------------------------------- | ----------- |
-| `.company(company_number)`  | `GET /company/:company_number`          | Retrieves a company profile. |
-| `.officers(company_number)` | `GET /company/:company_number/officers` | Retrieves a list of company officers. |
+| Client Method                     | Endpoint                                | Description |
+| --------------------------------- | --------------------------------------- | ----------- |
+| `.company(company_number)`        | `GET /company/:company_number`          | Retrieves a company profile. |
+| `.officers(company_number)`       | `GET /company/:company_number/officers` | Retrieves a list of company officers. |
+| `.search_companies(query, limit)` | `GET /search/companies`                 | Retrieves a list of companies that match the given query. |
```

Comment:
I'd probably call this `company_search` rather than `search_companies` - I feel like it meshes better with the other methods here. @timrogers thoughts?

Reply:
That looks better, indeed. I have changed the signature to match that.